config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id", to: "guest_rooms#create"
```

Comment:
This would be a tiny MVC issue with the path itself (as I explained above) but overall good work on these routes.